daml_dit_if/main/main.py

```diff
@@ -50,8 +50,9 @@ def load_integration_spec(config: 'Configuration') -> 'Optional[IntegrationRunti
 
 
 def create_network(url: str) -> 'Network':
+    token = optenv("DAML_LEDGER_TOKEN")
     network = Network()
-    network.set_config(url=url)
+    network.set_config(url=url, oath_token=token)
```

Comment:
Should this really be `oath_token` (as opposed to `oauth_token`?)

Reply:
Yes I believe you're correct, I must have erroneously changed it between testing and committing. I'll fix and retest when I'm back at my computer.

Reply:
Fixed this, you were correct